src/platform/visualc/config.h

```diff
@@ -42,6 +42,9 @@
 /* Define to 1 to enable FluidSynth MIDI synthesizer */
 #define C_FLUIDSYNTH 1
 
+/* Define to 1 to use Opus-encoded CD-audio tracks, requires opusfile */
+#define USE_OPUS 1
```

Comment:
Good job figuring it out… This means I am likely missing this define on `meson` branch as well - thanks!

Reply:
It was a bit of a nightmare because the track opening was failing but I couldn't figure out why (despite all the effort on the debugging front).
I eventually inspected the array-of-array `decoders` objects and saw the Opus callback object wasn't in the list when I iterated through them.